www/widgets/DemoExampleMaker.js

```diff
@@ -209,6 +213,10 @@ class DemoExampleMaker extends Widget {
 
   // ------------------------------
 
+  _previewStep (step) {
+    window.convo = new Convo({ content: this.$('[name="dem-s-text"]').value })
+  }
```

Comment:
i'd also add a conditional here to see if there are any "line numbers" specified to highlight, if so we should highlight those using `NNE.spotlight([numbers])` if not we should reset to no spotlight `NNE.spotlight(null)` 

in netnet the global `NNE` variable represents the main instance of the [netitor](https://github.com/netizenorg/netitor) ( "spotlight" is one of it's methods, u can check out all the [other methods in the netitor docs here](https://github.com/netizenorg/netitor#methods)).